index.html

```diff
@@ -57,28 +59,33 @@
   <script src="list.js"></script>
   <script src="chart.js"></script>
   <script src="addRate.js"></script>
+  <script src="singIn.js"></script>
 
   <nav class="navbar navbar-default">
-    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-      <ul class="nav navbar-nav links">
-        <li>
-          <p class="navbar-text "><a href="" class="navbar-link">Wszystkie</a></p>
-        </li>
-      </ul>
-
-      <ul class="nav navbar-nav">
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Inne <span class="caret"></span></a>
-          <ul class="dropdown-menu">
-
-          </ul>
-        </li>
-      </ul>
+    <div class="container-fluid">
+      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+        <ul class="nav navbar-nav links">
+          <li class="navbarLinks">
+            <p class="navbar-text "><a href="" class="navbar-link">Wszystkie</a></p>
+          </li>
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Inne <span class="caret"></span></a>
+            <ul class="dropdown-menu"></ul>
+          </li>
+        </ul>
+        <ul class="navbar-right">
+          <li class="singInSingUp navbar-text navbar-right">
```

Comment:
those classes aren't following any naming convention, please adapt (not blocking the merge, i know you will do that in later commits)